vocabularies/Org.OData.Core.V1.xml

```diff
@@ -457,6 +457,11 @@
         <Annotation Term="Core.Description" String="The qualified name of an action in scope." />
       </TypeDefinition>
 
+      <TypeDefinition Name="QualifiedBoundOperationName" UnderlyingType="Edm.String">
+        <Annotation Term="Core.Description" String="The qualified name of a bound action or function in scope." />
+        <Annotation Term="Core.LongDescription" String="Either the qualified name of an action, to indicate the single bound overload with the specified binding parameter type, the qualified name of a function, to indicate all bound overloads with the specified binding parameter type, or the qualified name of a function followed by parentheses containing a comma-separated list of parameter types, in the order of their definition, to identify a single function overload with the first (binding) parameter matching the specified parameter type." />

```

Comment:
```suggestion
        <Annotation Term="Core.LongDescription">
          <String>Either
- the qualified name of an action, to indicate the single bound overload with the specified binding parameter type, 
- the qualified name of a function, to indicate all bound overloads with the specified binding parameter type, or 
- the qualified name of a function followed by parentheses containing a comma-separated list of parameter types, in the order of their definition, to identify a single function overload with the first (binding) parameter matching the specified parameter type.
        </String>
        </Annotation>
```

So far we rely on XML retaining line-breaks within element text, combined with Markdown treatment of line-breaks.